components/TimeSeries.vue

```diff
@@ -0,0 +1,353 @@
+});
+const usePlotLines = props.seriesId === "hospitalised"; // https://mrc-ide.myjetbrains.com/youtrack/issue/JIDEA-118/
+const usePlotBands = props.seriesId === "hospitalised" || props.seriesId === "infect"; // https://mrc-ide.myjetbrains.com/youtrack/issue/JIDEA-118/
+// On zooming, the y-axis automatically rescales to the data (ignoring the plotLines). We want the plotLines
+// to remain visible, so we limit the y-axis' ability to rescale, by defining a minimum range.
+const minRange = computed(() => {
+  if (usePlotLines) {
+    return appStore.capacitiesData?.reduce((acc, { value }) => Math.max(acc, value), 0);
+  } else {
+    return undefined;
+  }
+});
+const capacitiesPlotLines = computed(() => {
+  const lines = Array<Highcharts.AxisPlotLinesOptions>();
+
+  if (!usePlotLines) {
+    return lines;
+  }
+
+  appStore.capacitiesData?.forEach(({ id, value }) => {
+    const capacityLabel = appStore.metadata?.results.capacities.find(({ id: capacityId }) => capacityId === id)?.label;
+
+    lines.push({
+      color: plotLinesColor,
+      label: {
+        text: `${capacityLabel}: ${value}`,
+        style: {
+          color: plotLinesColor,
+        },
+      },
+      width: 2,
+      value,
+      zIndex: 4, // Render plot line and label in front of the series line
+    });
+  });
+
+  return lines;
+});
+
+const interventionsPlotBands = computed(() => {
+  const bands = Array<Highcharts.AxisPlotBandsOptions>();
+
+  if (!usePlotBands) {
+    return bands;
+  }
+
+  appStore.interventionsData?.forEach(({ id, start, end }) => {
+    const interventionLabel = appStore.metadata?.results.interventions.find(({ id: interventionId }) => interventionId === id)?.label;
+    const text = `${interventionLabel ? `${interventionLabel} : ` : ""}Days ${start} to ${end}`;
+    bands.push({ from: start, to: end, color: plotBandsColor, label: { text } });
+  });
+
+  return bands;
+});
+
+const allCharts = computed(() => {
+  // Excludes undefineds. When a chart is destroyed, Highcharts.charts retains its index, but the value is undefined.
+  return Array.from(Highcharts.charts).filter(chart => chart) as (Highcharts.Chart)[];
+});
+
+const handleAccordionToggle = () => {
+  emit("toggleOpen");
+};
+
+const minChartHeight = minAccordionHeight - (2 * accordionBodyYPadding);
+const chartHeight = () => (containerHeightPx.value - (2 * accordionBodyYPadding));
+
+/**
+ * Synchronize tooltips and crosshairs between charts.
+ * Demo: https://www.highcharts.com/demo/highcharts/synchronized-charts
+ */
+const syncTooltipsAndCrosshairs = throttle(() => {
+  if (chart.hoverPoint) {
+    allCharts.value.forEach(({ series }) => {
+      // Get the point with the same x as the hovered point
+      const point = series[0].getValidPoints().find(({ x }) => x === chart.hoverPoint!.x);
+
+      if (point && point !== chart.hoverPoint) {
+        point.onMouseOver();
+      }
+    });
+  };
+}, 100, { leading: true });
```

Comment:
Is every TimeSeries doing this for every chart...?

Reply:
Yes, whenever any time series component's (singular) chart has a relevant mouse/touch event, we find the point that is hovered, work out what the equivalent points on the other charts are, and trigger those points on the other charts to show a tooltip.